db/migrate/20230920172608_create_create_join_table_hastags_tweets.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCreateJoinTableHastagsTweets < ActiveRecord::Migration[7.0]
+  def change
+    create_table :create_join_table_hastags_tweets do |t|
```

Comment:
This table could have been named `Taggins` or you could have use something more generic as `HastagTweets`